tests/libtest.sh

```diff
@@ -29,3 +29,14 @@ wipe_signatures() {
     wipefs -a $dev >> $LOGS 2>&1
   done
 }
+
+cleanup() {
+  local vg_name=$1
+  local devs=$2
+
+  vgremove -y $vg_name >> $LOGS 2>&1
+  remove_pvs "$devs"
+  remove_partitions "$devs"
+  clean_config_files
+  wipe_signatures "$devs"
+}
```

Comment:
Thanks for moving this common code in libtest.sh. Down the line I want to do more cleanup and move more common code in libtest.sh. That's for separate PR though.